src/AltinnCore/Designer/Views/Home/StartPage.cshtml

```diff
@@ -56,7 +56,7 @@
         <img src="~/designer/img/altinnStudio.png" />
       </div>
       <ul class="nav navbar-nav">
-        <li><a class="nav-link" href="/user/login">Logg inn <i class="fa fa-fw fa-user-circle" aria-hidden="true"></i></a></li>
+        <li><a class="nav-link" href="repos/user/login">Logg inn <i class="fa fa-fw fa-user-circle" aria-hidden="true"></i></a></li>
```

Comment:
Is it tested that we don't need a "/" before repos?

Reply:
I'm guessing that since all our tests have been successful and we have used the login button several times, that this is not an issue.

Reply:
Tested, it works without /